scripts/connectal-synth-axiddr3.tcl

```diff
@@ -4,4 +4,4 @@ source "$connectaldir/scripts/connectal-synth-ip.tcl"
 set prj_boardname $boardname
 if [string match "*g2" $boardname] {set prj_boardname [string trimright $boardname "g2"]}
 
```

Comment:
this number should be based on vivado version number or else we accept any version